orbbec_camera/component_descriptions/orbbec_camera_ob_camera_node.json

```diff
@@ -84,7 +114,148 @@
         "parameter_name": "align_mode",
         "parameter_type": "string",
         "default_value": "HW"
-      }
+      },
+      {
+        "display_name": "Alignment Target Stream",
+        "description": "Target stream to align with (COLOR or DEPTH)",
+        "parameter_name": "align_target_stream",
+        "parameter_type": "string",
+        "default_value": "COLOR",
+        "internal": true
+      },
+      {
+        "display_name": "Enable Threshold Filter",
+        "description": "Enable the depth threshold filter (if enabled, set threshold_filter_max and threshold_filter_min)",
+        "parameter_name": "enable_threshold_filter",
+        "parameter_type": "bool",
+        "default_value": "false"
+      },
+      {
+        "display_name": "Threshold Filter Max",
+        "description": "Max value of the threshold filter [mm]. Values in [0, 16000].",
```

Comment:
```suggestion
        "description": "Max value of the threshold filter (between 0 and 16000) [mm].",
```

Reply:
I would propose more something like this for all values that follow, i.e.
"description (between X and Y) [unit if applicable]"
Open to other suggestions